### PR TITLE
[Azure Communication Services] Phone Number Admin - Add CHANGELOG for Nov release

### PR DESCRIPTION
**Pull request by HeliWang**

### CODE REVIEW

**Review by ankitarorabit — APPROVED** [review]

File: sdk/communication/azure-communication-administration/CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,28 @@
 # Release History
+## 1.0.0b3 (Unreleased) 
+
+**Breaking Changes**
+
+- `PhoneNumberSearch` has been renamed to `PhoneNumberReservation`
+- `search_id` has been renamed to `reservation_id` in `PhoneNumberReservation`
+- `get_search_by_id` has been renamed to `get_reservation_by_id`
```
Comment (RezaJooyandeh) [inline sdk/communication/azure-communication-administration/CHANGELOG.md:8]: I think we should say what is the type in which this method is renamed. So maybe we can add a section like

### `PhoneNumberSearch` renamed to `PhoneNumberReservation`

### `FooClient`
- `get_search_by_id` has been renamed to `get_reservation_by_id`
- `create_search` has been renamed to `begin_reserve_phone_numbers`
-  `begin_release_phone_numbers` now returns LROPoller[PhoneNumberRelease]

### `PhoneNumberReservation`
- `search_id` is renamed to `reservation_id`

Comment (HeliWang) [reply sdk/communication/azure-communication-administration/CHANGELOG.md:12]: Thanks. I have updated my PR to have the additional section.

File: sdk/communication/azure-communication-administration/CHANGELOG.md [hunk 2]
```diff
@@ -1,4 +1,32 @@
...
+##### `PhoneNumberSearch` renamed to `PhoneNumberReservation`
+
+##### `PhoneNumberReservation`
+- `search_id` has been renamed to `reservation_id`
+
+##### `PhoneNumberAdministrationClient`
+- `get_search_by_id` has been renamed to `get_reservation_by_id`
+- `create_search` has been renamed to `begin_reserve_phone_numbers`
+-  `begin_reserve_phone_numbers` now takes either `options`, or `continuation_token` keywords as input
+-  `begin_reserve_phone_numbers` now returns LROPoller[PhoneNumberReservation]
```
Comment (xiangyan99) [inline sdk/communication/azure-communication-administration/CHANGELOG.md:15]: maybe `LROPoller[PhoneNumberReservation]`?

File: sdk/communication/azure-communication-administration/CHANGELOG.md [hunk 3]
```diff
@@ -1,4 +1,35 @@
...
+-  `begin_release_phone_numbers` now returns LROPoller[PhoneNumberRelease]
+- `purchase_search` has been renamed to `begin_purchase_reservation`
+-  `begin_purchase_reservation` now takes either reservation_id, or `continuation_token` keywords as input
+-  `begin_purchase_reservation` now returns LROPoller[PurchaseReservationPolling]
+- `cancel_search` has been renamed to `cancel_reservation`
+
+**New Features**
+
+##### Sync `PhoneNumberAdministrationClient`
+- Add long run operation polling method `ReservePhoneNumberPolling`,`PurchaseReservationPolling`,
```
Comment (RezaJooyandeh) [inline sdk/communication/azure-communication-administration/CHANGELOG.md:27]: nit: we should consistently use "." at the end of all sentences

File: sdk/communication/azure-communication-administration/CHANGELOG.md (hunk 2 above)
Comment (HeliWang) [reply sdk/communication/azure-communication-administration/CHANGELOG.md]: Thanks. Back-quote has been added to missed places.

File: sdk/communication/azure-communication-administration/CHANGELOG.md (hunk 3 above)
Comment (HeliWang) [reply sdk/communication/azure-communication-administration/CHANGELOG.md:27]: Thanks. Fixed.

**Pull request merged**